_posts/2020-03-07-Team-Retreat.md

```diff
@@ -0,0 +1,69 @@
+
+Therefore, it was definitely interesting for us to bring the team together and give everyone an opportunity to interact. After all, we spend a great part of our day in touch. We might as well put a "real world" face to the names.
+
+# The Planning
+
+When putting together a team retreat, it is important to plan it well. What are the best dates? How about a location? Team members will be travelling from their home countries to the retreat destination to spend a few days with the team, it is important to take distance into consideration.
+
+Additionally, it is important to plan for visas. What if a team mate's visa is denied? What if the process takes too long? One lesson we learnt is it is always better to plan ahead. At OmbuLabs, our goal now is to announce the retreat dates and destination a year in advance. This way, the team has a chance to get all necessary documentation sorted.
+
+It is also important to have a retreat point of contact. Someone who will be in charge of planning and can answer questions regarding flights, accommodation, schedule, documentation, etc. This way, team members aren't lost seeking information.
+
+# The Agenda
+
+Once the retreat is planned and everything is booked, we need an agenda for the event. After all, we're having a once in a year opportunity to bring everyone to the same location.
+
+We like to open our retreats with a talk from our CEO on the company's vision and mission. This way, we all know where we're going.
+
+It is also important to plan your agenda carefully. A good way to get to great topics is to ask yourself "what challenges are we facing right now? And, of these, which ones impact us the most?". In a team retreat we have the opportunity to bring all heads together in the same room. It is a unique opportunity to foster discussion face-to-face, without the reservations we all have when interacting online. Therefore, make it matter.
+
+At our team retreat, we talked about company culture, discussed our mentorship programme, and gathered ideas to make our systems and processes better. 
+
+However, a good agenda must also include fun activities. A system we found effective was to intercalate presentations and discussions with fun team activities. This way, the day doesn't get boring and we can foster better discussions.
+
+# The Activities
+
+When planning activities in a team retreat, it is important to think of the value they bring. We want activities that will bring the team together while being entertaining, that will serve a clear purpose while being fun and enjoyable.
+
+Some activities we did include:
+
+- [SpaceTeam](http://playspaceteam.com)
+- [Escape Room in a Box](http://www.escaperoominabox.com/en-us)
+- Ask Me Anything
+- Two Truths and a Lie
```

Comment:
@ABizzinotto Maybe you could include a photo in here?

Reply:
Done